.github/workflows/workflow.yml

```diff
@@ -7,11 +7,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 18.x]
+        node-version: [12.x, 14.x, 16.x, 18.x, 20.x]
```

Comment:
`12.x` is no longer maintained. Should we remove it? 
https://github.com/nodejs/release#release-schedule
<img width="942" alt="image" src="https://user-images.githubusercontent.com/194400/234801909-6c434d3d-8236-4f07-a9f6-9394830d32ec.png">

Reply:
Thanks for the review.
I have removed 12.x as it is already EOL.
14.x is also soon to be EOL, so I removed it.